src/_guides/language/evolution.md

```diff
@@ -180,131 +183,190 @@ turn into edit-time analysis errors.
 
 In Dart 2.12, **[Dart FFI][]** graduated from beta to the stable channel.
 
-### Dart 2.13
+### Dart 2.10
+_Released 1 October 2020_
+| [Dart 2.10 announcement](https://medium.com/dartlang/announcing-dart-2-10-350823952bd5)
 
-Dart 2.13 expanded support for **[type aliases][]** (`typedef`),
-which used to work only for function types
-but now work for any type.
-You can use the new name created with a type alias
-anywhere the original type could be used.
+Dart 2.10 added no new features to the Dart language.
+It did add an expanded [`dart` tool][dart-tool] similar to the
+Flutter SDK's [`flutter` tool][].
 
-Dart 2.13 also improved the struct support in **[Dart FFI][]**,
-adding support for inline arrays and packed structs.
+### Dart 2.9
 
-### Dart 2.14
+Dart 2.9 added no new features to the Dart language.
 
-Dart 2.14 added the unsigned shift operator (`>>>`),
-also known as _triple-shift_.
-This new operator is similar to `>>`,
-except that it always fills the most significant bits with zeros.
-For more information, see the
-[bitwise and shift operator][] section of the language tour.
+### Dart 2.8
+_Released 6 May 2020_
+| [Dart 2.8 announcement](https://medium.com/dartlang/announcing-dart-2-8-7750918db0a)
 
-[bitwise and shift operator]: /language/operators#bitwise-and-shift-operators
+Dart 2.8 didn't add any features to the Dart language. It did
+contain a number of preparatory [breaking changes][2.8 breaking changes]
+to improve nullability-related usability and performance for [null safety][].
 
-Dart 2.14 also removed some restrictions on type arguments.
-You can now pass type arguments to annotations,
-and you can use a generic function type as a type argument.
-All of the following code was invalid before 2.14,
-but is now allowed:
+It improved the performance of the `pub` tool and
+added the [`pub outdated`][] command.
+
```

Comment:
Remove this tool mention as well to be consistent with the focused direction I mentioned before.

We don't mention tool changes for the other releases.


```suggestion
```